rollup.config.mjs

```diff
@@ -61,6 +62,41 @@ export default {
       extract: "dist/index.css",
       extensions: [".sass", ".scss", ".css"],
     }),
+    {
```

Comment:
When you compile Sass, it generates a CSS map (sourcemap) file that helps to track the positions of CSS and original Sass files. However, if the order of text in the CSS file changes, the sourcemap may no longer be valid.

Regarding whether there will be any impact on this issue (whether it's safe to remove it), personally, I think setting sourcemap: false shouldn't cause significant problems.


```js
postcss({
  // ...
  sourceMap: false,
})
```

Reply:
Hmm.. let's remove it

Reply:
interesting. Ill look into this maybe next week due to other priorities

Reply:
@bang9 I asked chatGPT and below is answer

> If you add a custom plugin like 'move-imports-to-top', it won't automatically handle sourcemaps for you. Sourcemaps are typically handled by the plugins provided by Rollup or other build tools.

It also suggested me to handle mapping manually using
`import { SourceMapConsumer, SourceMapGenerator } from 'source-map';`

Do you think we should try this? or just set sourceMap: false?

Reply:
`sourceMap: false` let's go~